pkg/models/notifyupdate.go

```diff
@@ -0,0 +1,13 @@
+package models
```

Comment:
Why separating this types to a new file. I think it's better to have them in registration.go as they are related.